tests/legacy/test_lib.py

```diff
@@ -137,33 +110,19 @@ def create_test_cache_folder():
 # pylint: disable=no-member
 app.logger.init_logging(False)
 
-
-# =================
-#  dummy functions
-# =================
-def _dummy_save_config():
-    """
-    Override the SickBeard save_config which gets called during a db upgrade.
-
-    :return: True
-    """
-    return True
-
-# this overrides the SickBeard save_config which gets called during a db upgrade
+# this overrides the app save_config which gets called during a db upgrade
 # this might be considered a hack
-main_db.app.save_config = _dummy_save_config
+main_db.app.save_config = lambda: True
```

Comment:
@ratoaq2 why we use from main_db ?  I have this issue related to this: https://github.com/pymedusa/SickRage/pull/1158#discussion_r79642549